git-hub

```diff
@@ -1330,7 +1330,7 @@ class PullUtil (IssueUtil):
 
 	# Perform a validated git fetch
 	@classmethod
-	def git_fetch(cls, url, ref, urltype=config.urltype):
+	def git_fetch(cls, url, ref, urltype):
```

Comment:
Actually, is always using `config.urltype`, so we can just not pass it as an argument and use it in the body (by the time it is used, it is defined).